packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -107,7 +107,7 @@ export default class LocusInfo extends EventsScope {
         meeting.meetingRequest
           .getFullLocus({
             desync: true,
-            locusUrl: meeting.locusUrl,
+            locusUrl: locus.url ? locus.url : meeting.locusUrl,
```

Comment:
small suggestion: can just simplify it as `locus.url || meeting.locusUrl`

Reply:
Good